server/graphql/invoice/src/mutations/outbound_return/update_name.rs

```diff
@@ -0,0 +1,381 @@
+use async_graphql::*;
+
+use graphql_core::{
+    simple_generic_errors::{OtherPartyNotASupplier, OtherPartyNotVisible, RecordNotFound},
+    standard_graphql_error::{validate_auth, StandardGraphqlError},
+    ContextExt,
+};
+use graphql_types::types::InvoiceNode;
+use repository::Invoice;
+use service::auth::{Resource, ResourceAccessRequest};
+use service::invoice::outbound_return::update_name::{
+    UpdateOutboundReturnName as ServiceInput, UpdateOutboundReturnNameError as ServiceError,
+};
+
+use crate::mutations::outbound_shipment::error::InvoiceIsNotEditable;
+
+#[derive(InputObject)]
+#[graphql(name = "UpdateOutboundReturnNameInput")]
+pub struct UpdateNameInput {
+    pub id: String,
+    other_party_id: Option<String>,
+}
+
+#[derive(SimpleObject)]
+#[graphql(name = "UpdateOutboundReturnNameError")]
+pub struct UpdateNameError {
+    pub error: UpdateReturnNameErrorInterface,
+}
+
+#[derive(Union)]
+#[graphql(name = "UpdateOutboundReturnNameResponse")]
+pub enum UpdateNameResponse {
```

Comment:
NIT, but I think `updateOtherParty` would have been clearer naming to me. I know it sets the name_link_id at the repository layer, but everywhere else we call it other party... I thought we were naming the returns like instead of referring to them by their invoice numbers for a sec LOL